src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,29 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
+import mate.academy.hibernate.relations.exeptions.DataProcessingException;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
+import org.hibernate.SessionFactory;
 
 public class ActorServiceImpl implements ActorService {
+    private final SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+    private final ActorDao actorDao = new ActorDaoImpl(sessionFactory);
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        Optional<Actor> actorOptional = actorDao.get(id);
+        if (actorOptional.isEmpty()) {
+            throw new DataProcessingException("Actor with id = " + id + " no exist.");
+        }
```

Comment:
1. Please use orElseThrow method instead of if block
2. Please chose another exception class here. DataProcessingException should mean that we have problem with DB and cannot provide result. In this case everything is fine with DB, we just don't have such record, so NoSuchElementException or EntityNotFoundException will be much more appropriate